.prettierrc.json

```diff
@@ -1,4 +1 @@
-{
-  "singleQuote": true,
-  "trailingComma": "es5"
-}
+{}
```

Comment:
Please revert this, as vscode otherwise will fallback to older default 😕 Not sure if this is fixed in latest vscode. So please verify before

Reply:
Reverted. 😉